database-service/tests/test_migrations.py

```diff
@@ -1,16 +1,22 @@
-import os
-import sh
+def test_shopozor_structural_migrations_can_be_applied(hasura_client, app_root_folder):
+    # Given I have structural migrations
+    project_folder = app_root_folder
 
+    # When I apply them
+    exit_code = hasura_client.apply_migrations(project_folder)
 
-def test_migrations_can_be_applied_without_errors(hasura_endpoint):
-    # Given I am in the right location
-    assert os.path.exists('migrations')
-    assert os.path.isfile('config.yaml')
+    # Then I get no error
+    assert 0 == exit_code
 
-    # When I apply the migrations
-    cmd = sh.Command('hasura')
-    result = cmd('migrate', 'apply', '--endpoint',
-                 hasura_endpoint, '--skip-update-check')
 
-    # Then I get no errors
-    assert 0 == result.exit_code
+def test_fixtures_migrations_can_be_applied(hasura_client, app_root_folder, small_fixtures):
+    # Given I've applied structural migrations
+    structural_project_folder = app_root_folder
+    exit_code = hasura_client.apply_migrations(structural_project_folder)
+    assert 0 == exit_code
+
+    # When I apply the fixtures migrations
+    exit_code = hasura_client.apply_migrations(small_fixtures)
+
+    # Then I get no error
+    assert 0 == exit_code
```

Comment:
The two tests officially verify that migrations can be applied. Rolling them back is done in the fixtures clean up (cf. fixtures with `yield` statement in `conftest.py`). If fixtures clean up fails, then the tests fail too.